content/en/docs/setup/platform-setup/openshift/index.md

```diff
@@ -74,3 +86,19 @@ $ oc ex config patch master-config.yaml.prepatch -p "$(cat master-config.patch)"
 $ master-restart api
 $ master-restart controllers
 {{< /text >}}
+
+## Bookinfo
+
+The Bookinfo application specifies a user ID of 1337, which is not allowed by default in OpenShift. To allow this user ID to be used, execute the following commands. Replace `-n bookinfo` with the appropriate namespace.
```

Comment:
This is very surprising. UID 1337 is the one running Envoy in the `istio-proxy` sidecar container. Anything running with that UID bypasses the sidecar proxy, so Bookinfo certainly doesn't run with that UID.
So you probably meant something like:
```suggestion
The Istio sidecar injected into each pod runs with user ID 1337, which is not allowed by default in OpenShift. To allow this user ID to be used, execute the following commands. Replace `-n bookinfo` with the appropriate namespace.
```

Reply:
Thanks.